src/components/checkbox/checkbox.style.ts

```diff
@@ -137,9 +137,6 @@ const StyledCheckbox = styled.div<StyledCheckboxProps>`
 
       ${StyledFieldHelp} {
         margin-left: 24px;
-        padding-left: ${labelSpacing === 1
```

Comment:
question: why has this been removed?

I take it it's no longer needed it just stood out with it being a RTL conversion that we're making a styling change here

Reply:
I removed this as I noticed it was already setting the labelSpacing padding in line [133](https://github.com/Sage/carbon/blob/42323ee6b1677d9317c7dcaa485eb9b53a0f93f8/src/components/checkbox/checkbox.style.ts#L113) and this was just copied over into the large size styles so it made no difference to remove this. For coverage we would have also needed another test which I thought was pointless.

Reply:
Okay, if it was just style bloat happy to remove it. I've just moved the PR to `ready for review` so we can just make sure Chromatic has no issues